create-container.c

```diff
@@ -474,15 +480,18 @@ int main(int argc, char* argv[])
 	while (1) {
 		int opt;
 #ifdef _AIX
-		opt = getopt(argc, argv, "?hvdw:a:b:c:p:q:r:A:B:C:P:Q:R:L:I:o:O:f:F:l:0:1:2:");
+		opt = getopt(argc, argv, "?hvdw:a:b:c:p:q:r:A:B:C:P:Q:R:L:I:o:O:f:F:l:0:1:2:S:");
 #else
 		opt = getopt_long(argc, argv,
-				"hvdw:a:b:c:p:q:r:A:B:C:P:Q:R:L:I:o:O:f:F:l:0:1:2:", opts,
+				"hvdw:a:b:c:p:q:r:A:B:C:P:Q:R:L:I:o:O:f:F:l:0:1:2:S:", opts,
 				NULL);
 #endif
 		if (opt == -1)
 			break;
 
+        // Set the values for optional args
+        params.security_version = 0;
```

Comment:
Since this code is looping on arguments should the default be set outside?  Otherwise I don't think this works if -S is not the last option